src/SFA.DAS.SharedOuterApi/InnerApi/Requests/GetPingRequest.cs

```diff
@@ -0,0 +1,10 @@
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.InnerApi.Requests
```

Comment:
```suggestion
namespace SFA.DAS.SharedOuterApi.InnerApi.Requests
```
this might break some references so best do it using resharper